packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -2775,29 +2777,56 @@ export const audiusBackend = ({
     }
   }
 
+  /**
+   * Returns whether the current user is subscribed to userId.
+   * @returns Boolean
```

Comment:
with typescript, the @returns js-doc is not needed, we can either rely on the implicit return type, or you can explicitly type it with 
```
async function getUserSubscribed(userId: ID): Promise<boolean> {
```